### PR TITLE
Clear events

### PR DESCRIPTION
**Pull request by aleks-tpom6oh**

### CODE REVIEW

File: IosAnalyticsDebugger/Classes/eventslistscreen/EventsListScreenViewController.xib [hunk 1]
```diff
@@ -54,21 +54,32 @@
                             <color key="textColor" red="0.60392156860000001" green="0.60392156860000001" blue="0.60392156860000001" alpha="1" colorSpace="calibratedRGB"/>
                             <nil key="highlightedColor"/>
                         </label>
+                        <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="system" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="e4A-OW-CdX">
+                            <rect key="frame" x="300" y="8" width="64" height="32"/>
```
Comment (logason) [inline IosAnalyticsDebugger/Classes/eventslistscreen/EventsListScreenViewController.xib]: Can we make the button slightly smaller and align better with our design spec? 

Perhaps something in this direction: https://www.figma.com/file/IcXFkmJ8atZKdBAFX5a6xEuG/Design-System?type=design&node-id=1968%3A86&mode=design&t=fgPSKnWk1NjEEPMu-1
Screenshot 2023-10-26 at 16 15 52

File: IosAnalyticsDebugger/Classes/eventslistscreen/EventsListScreenViewController.m [hunk 2]
```diff
@@ -23,6 +23,10 @@ @interface EventsListScreenViewController ()
 @end
 
 @implementation EventsListScreenViewController
+- (IBAction)onClearButtonClick:(id)sender {
+    [AnalyticsDebugger.events removeAllObjects];
```
Comment (logason) [inline IosAnalyticsDebugger/Classes/eventslistscreen/EventsListScreenViewController.m:27]: Nice, was the `removeAllObjects` already implemented?

Comment (aleks-tpom6oh) [reply IosAnalyticsDebugger/Classes/eventslistscreen/EventsListScreenViewController.m:27]: Yes, it's conveniently available in standard library 😄

File: IosAnalyticsDebugger/Classes/eventslistscreen/EventsListScreenViewController.xib (hunk 1 above)
Comment (aleks-tpom6oh) [reply IosAnalyticsDebugger/Classes/eventslistscreen/EventsListScreenViewController.xib]: Improved

Screenshot 2023-10-27 at 20 39 03

Comment (logason) [reply IosAnalyticsDebugger/Classes/eventslistscreen/EventsListScreenViewController.xib]: I'd make it slightly smaller, and the text semibold 💅

**Review by logason — APPROVED** [review]
One minor comment on the layout, otherwise LGTM!

**Pull request merged**